addons/hr_timesheet/models/ir_http.py

```diff
@@ -24,6 +24,9 @@ def session_info(self):
                         company.timesheet_encode_uom_id,
                         round=False
                     ),
+                    "timesheet_cost_factor": company.project_time_mode_id._compute_cost(
```

Comment:
is that really necessary to add this in the session_info? can't you just make an rpc?